### DISCUSSION TITLE
Not clear how to use websocket/uberduck.ai

### DISCUSSION THREAD

**Discussion by CRYPTark01** [karashiiro/TextToTalk#110] [Q&A] [answered]
I would love to use something other than windows voices and amazon AWS scares me by not being able to understand what will be charged after 12 months. I signed up for Uberduck.ai and got an API key and secret but don't see how to connect it to the plugin. Any guide?

**Comment by karashiiro** [comment]
There is currently no Uberduck.ai support in the plugin, but there is a feature request for it (#74). I'm going to work on that after handling some Universalis feature requests.

**Reply by CRYPTark01** [reply]
Ah, okay well honestly anything you’re implementing that will allow for better voice beyond Amazon Polly I’ll use. I’m just worried about Amazon Polly because you cant simply remove your credit card so I’d be worried what might happen if I forget after twelve months of free and then bam start running tons of calls to their service generating a big bill. Whether that’s actually something to be concerned about I’m not sure. So if you’re working on some other Ai TTS api integration that sounds good to me.

**Comment by karashiiro** [comment, marked as answer]
Uberduck support is implemented as of v1.15.
